vmhost/vmhooks/smallIntOps.go

```diff
@@ -270,11 +274,15 @@ func v1_4_smallIntStorageLoadSigned(context unsafe.Pointer, keyOffset int32, key
 		return 0
 	}
 
-	data, usedCache, err := storage.GetStorage(key)
+	data, trieDepth, usedCache, err := storage.GetStorage(key)
 	if vmhost.WithFault(err, context, runtime.BaseOpsErrorShouldFailExecution()) {
 		return 0
 	}
-	storage.UseGasForStorageLoad(smallIntStorageLoadSignedName, metering.GasSchedule().BaseOpsAPICost.Int64StorageLoad, usedCache)
+	err = storage.UseGasForStorageLoad(smallIntStorageLoadSignedName, int64(trieDepth), metering.GasSchedule().BaseOpsAPICost.Int64StorageLoad, usedCache)
+	if err != nil {
```

Comment:
please refactor this to resemble what we have in v1.5:
```GO
err = storage.UseGasForStorageLoad(smallIntStorageLoadSignedName, trieDepth, metering.GasSchedule().BaseOpsAPICost.Int64StorageLoad, usedCache)
if context.WithFault(err, runtime.BaseOpsErrorShouldFailExecution()) {
    return -1
}
```

Reply:
Done.